### DISCUSSION TITLE
创建完成之后提示这个窗口，是我哪部分输入错了嘛

### DISCUSSION THREAD

**Discussion by GPT-Bunny** [tw93/Pake#484] [Q&A]
![8991687595639_ pic](https://github.com/tw93/Pake/assets/133673357/aef8d528-a109-44f2-afec-f9d6836b52f8)
![9001687595993_ pic](https://github.com/tw93/Pake/assets/133673357/d6623837-4c8e-4413-93c4-4e066d4ada34)

**Comment by tw93** [comment]
名字不对，直接试试单个的词语
